lib/napa/cli/templates/new/.ruby-version.tt

```diff
@@ -1 +1 @@
-2.0.0
\ No newline at end of file
+<%= RUBY_VERSION %>
```

Comment:
@shaqq: this file is normally used by `rbenv` and many version managers to change to the specified Ruby version

I think it also makes sense, if you generate this file, to do it with the executing ruby version